cmake/packaging_test_project/edsl_tests.cc

```diff
@@ -0,0 +1,2114 @@
+// Copyright 2020 Intel Corporation
```

Comment:
This file seems to be a duplicate of the existing `edsl_tests.cc`, are there any differences here? I think having a duplicate file makes it much harder to maintain over time.

Reply:
The only difference is the main() function at the end so it can be compiled into an executable directly (to keep the CMakeLists of the test project minimal). If there is a good way to reference the actual `edsl_tests.cc` file from the test project and compile it into an executable (namely, without the use of the `pml_cc_test` function).